pkg/controller/container-runtime-config/helpers.go

```diff
@@ -281,6 +282,22 @@ func createCRIODropinFiles(cfg *mcfgv1.ContainerRuntimeConfig) []generatedConfig
 	return generatedConfigFileList
 }
 
+// updateSearchRegistriesConfig gets the ContainerRuntimeSearchRegistries data from the Image CRD
+// and creates a drop-in file for it at /etc/containers/registries.conf.d
+func updateSearchRegistriesConfig(searchRegs []string) []generatedConfigFile {
+	var (
```

Comment:
minor nit: could just change below line to `generatedConfigFileList, err :=` and skip the extra definition lines

Reply:
The `addTOMLgeneratedConfigFile` function needs a `[]generatedConfigFile` to be passed in, that is why I have the definition earlier on.

Reply:
ah I missed that, :+1: